src/main/java/org/linkeddatafragments/solver/ReorderTransformationLDF.java

```diff
@@ -61,7 +61,6 @@ private void initializeMatcher() {
         matcher.addPattern(new Pattern(numTriples, VAR, VAR, VAR));     // ???
     }
 
-    @Override
```

Comment:
Good point... it is gone, but then it is not clear when this method would be used... do you have a test which relies on the function of this method? But honestly, I have no clue at this moment how the new class replaces they weight functionality...